superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -32,6 +32,8 @@ import {
 } from 'src/components/MessageToasts/actions';
 import { getClientErrorObject } from 'src/utils/getClientErrorObject';
 import COMMON_ERR_MESSAGES from 'src/utils/errorMessages';
+import { LOG_ACTIONS_SQLLAB_FETCH_QUERY } from 'src/logger/LogUtils';
```

Comment:
Could we make the event type more explicit like `LOG_ACTIONS_SQLLAB_FAILED_QUERY`? This would help when interpreting the event and also remove the need for the `has_err` attribute.

Reply:
I followed this way as `LOG_ACTIONS_RENDER_CHART` implemented using `has_err` attribute but I agree with your idea to have a dedicated logging type name.
Let me update accordingly